sync/rpc/server.go

```diff
@@ -308,3 +328,91 @@ func (*SyncAgentServer) BackupRestoreIncrementally(ctx context.Context, req *Bac
 	logrus.Infof("Done running %s %v", "sbackup", cmd.Args)
 	return &Empty{}, nil
 }
+
+//The following APIs implement the CRUD interface for the backup object
```

Comment:
Space after `//`. And this should fail the `go lint`? Because the comment of one function should start with `// BackupCreate xxxx`.